src/molecules/sidebar/index.tsx

```diff
@@ -49,15 +49,28 @@ export const Sidebar = ({
 
   return (
     <div>
-      <Drawer open={isOpen} onClose={onToggle}>
+      <Drawer
```

Comment:
Don't hard code the color for the background of the sidebar. The color should update on every color change when a user does from the ThemePicker .